org.bitcoincore.bitcoin-qt.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+
+  <id>org.bitcoincore.bitcoin-qt</id>
+  <name>Bitcoin Core</name>
+  <project_license>MIT</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+
+  <summary>
+    Bitcoin Core connects to the Bitcoin P2P network
+  </summary>
+
+  <url type="homepage">
+    https://bitcoincore.org/
+  </url>
+
+  <launchable type="desktop-id">
+    org.bitcoincore.bitcoin-qt.desktop
+  </launchable>
+
+  <description>
+    <p>
+    Bitcoin is an experimental digital currency that enables instant
```

Comment:
ACK: This matches the description in our [README.md](https://github.com/bitcoin/bitcoin#what-is-bitcoin).

Reply:
Is that something I should look at?